scripts/devops_tasks/setup_execute_tests.py

```diff
@@ -276,13 +276,14 @@ def execute_global_install_and_test(
     )
 
     parser.add_argument(
-        "--omit-management",
-        dest="omit_management",
-        default=False,
-        action="store_true",
-        help="Flag that indicates to omit any management packages except any management packages that should not be filtered. for e.g azure-mgmt-core",
+        "--filter-type",
```

Comment:
I was about to suggest updates to `build-test.yml` but I realize we don't make use of the filter type for it :P